### PR TITLE
[no-jira] Fix ts build

### PR DESCRIPTION
**Pull request by szymonmiks-netguru**
### Desc:
- fixed typescript build process
- update packages
- update README

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,7 +12,8 @@
     "coveralls": "cat ./coverage/lcov.info | coveralls",
     "lint": "tslint {src,test}/**/*.ts",
     "prebuild": "yarn clean",
-    "build": "tsc"
+    "build": "tsc -p tsconfig.json",
+    "prepublish": "tsc -p tsconfig.json"
```
Comment (AmirAlOmari) [inline package.json]: Here could be just `"npm run build"` to keep consistent 👍 
```suggestion
    "prepublish": "npm run build"
```

**Review by AmirAlOmari — APPROVED** [review]
Great, thanks @szymonmiks-netguru  🚀

File: package.json (hunk 1 above)
Comment (szymonmiks-netguru) [reply package.json]: Ok cool! I wasn't sure if `npm run build` will work inside another `npm` script! Great will do!

**Review by gezpage — APPROVED** [review]
Published version `0.3.5` to npmjs https://www.npmjs.com/package/@lifeworks/serverless-stack-output

**Pull request merged**